gammapy/image/plotting.py

```diff
@@ -434,3 +426,174 @@ def _panel_parameters(npanels, center, fov,
+
+    f = TransformedBbox(psf_box, ff._ax1.transData)
+    g = TransformedBbox(f, ff._figure.transFigure.inverted())
+    p1, p2 = g.get_points()
+    rect = [p1[0], p1[1], p2[0] - p1[0], p2[1] - p1[1]]
+
+    # WCSAxes should be used here
+    psf = ff._figure.add_axes(rect)
+
+    for spline in psf.spines.values():
+        spline.set_edgecolor(color)
+        spline.set_linewidth(linewidth)
+
+    psf.xaxis.set_ticks([])
+    psf.yaxis.set_ticks([])
+
+    psf.imshow(psf_image.data, **kwargs)
+    xc, yc = psf_position
+    wc = width / abs(h['CDELT1']) / 2.
+    hc = height / abs(h['CDELT2']) / 2.
+    psf.set_xlim(xc - wc, xc + wc)
+    psf.set_ylim(yc - hc, yc + hc)
+    return psf
+
+
+def world2fig(ff, x, y):
+    """
+    Helper function to convert world to figure coordinates.
+
+    Parameters
+    ----------
+    ff : `~aplpy.FITSFigure`
+        `~aplpy.FITSFigure` instance.
+    x : ndarray
+        Array of x coordinates.
+    y : ndarray
+        Array of y coordinates.
+
+    Returns
+    -------
+    coordsf : tuple
+        Figure coordinates as tuple (xfig, yfig) of arrays.
+    """
+    # Convert world to pixel coordinates
+    xp, yp = ff.world2pixel(x, y)
+
+    # Pixel to Axes coordinates
+    coordsa = ff._ax1.transData.transform(zip(xp, yp))
+
+    # Axes to figure coordinates
+    coordsf = ff._figure.transFigure.inverted().transform(coordsa)
+    return coordsf[:, 0], coordsf[:, 1]
+
+
+def grayify_colormap(cmap, mode='skimage'):
+    """
+    Return a grayscale version a the colormap.
+
+    The grayscale conversion of the colormap is bases on perceived luminance of
+    the colors. For the conversion either the `~skimage.color.rgb2gray` or a
+    generic method called ``hsp`` [1]_ can be used. The code is loosely based
+    on [2]_.
+
+
+    Parameters
+    ----------
+    cmap : str or `~matplotlib.colors.Colormap`
+        Colormap name or instance.
+    mode : str
+        Grayscale conversion method. Either ``skimage`` or ``hsp``.
+
+    References
+    ----------
+
+    .. [1] Darel Rex Finley, "HSP Color Model - Alternative to HSV (HSB) and HSL"
+       http://alienryderflex.com/hsp.html
+
+    .. [2] Jake VanderPlas, "How Bad Is Your Colormap?"
+       https://jakevdp.github.io/blog/2014/10/16/how-bad-is-your-colormap/
+    """
+    import matplotlib.pyplot as plt
+    cmap = plt.cm.get_cmap(cmap)
+    colors = cmap(np.arange(cmap.N))
+
+    if mode == 'skimage':
+        from skimage.color import rgb2gray
+        luminance = rgb2gray(np.array([colors]))
+        colors[:, :3] = luminance[0][:, np.newaxis]
+    elif mode == 'hsp':
+            RGB_weight = [0.299, 0.587, 0.114]
+            luminance = np.sqrt(np.dot(colors[:, :3] ** 2, RGB_weight))
+            colors[:, :3] = luminance[:, np.newaxis]
+    else:
+        raise ValueError('Not a valid grayscale conversion mode.')
+    return cmap.from_list(cmap.name + "_grayscale", colors, cmap.N)
+
+
+def analyze_colormap(cmap, **kwargs):
```

Comment:
Maybe rename `analyze_colormap` to `show_colormap` or `illustrate_colormap`?
It doesn't do any analysis itself, just plots things.